metaflow/plugins/kfp/kfp_metaflow_step.py

```diff
@@ -308,6 +306,11 @@ def kfp_metaflow_step(
     preceding_component_outputs_env: Dict[str, str] = {
         field: kwargs[field] for field in preceding_component_outputs
     }
+    if volume_dir:
+        clean_volume_cmd: str = f"rm -rf {os.path.join(volume_dir, '*')}"
+    else:
+        clean_volume_cmd: str = "true"
```

Comment:
This isn't the level that `clean_volume_cmd` should be, it should be in `def _command`

Reply:
As outlined even in https://github.com/zillow/metaflow/pull/137#discussion_r749882134

Reply:
Sounds good, both `clean_volume_cmd` and `task_id_template` created within `_command` right on top of the command they're used in for readability.